native/src/components/AppStateListener.ts

```diff
@@ -22,7 +20,7 @@ const AppStateListener = (): ReactElement | null => {
     }
   }
 
-  useEffect(() => AppState.addEventListener('change', handleAppStateChange).remove, [])
+  useAppStateListener(handleAppStateChange)
```

Comment:
can you explain this change, please

Reply:
I created a custom hook to make it reusable/tidy up the TtsContainer. I just reused it here in this place as well.